### PR TITLE
quicksort blog

### PR DESCRIPTION
**Pull request by stacyburris**
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: javascript/code-challenges/quickSort/BLOG.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+3. Start the right pointer at last element of the array.
+4. Compare the element pointing with left pointer and if it is less than the pivot element, then move the left pointer to the right (add 1 to the left index). Continue this until left side element is greater than or equal to the pivot element.
+5. Compare the element pointing with right pointer and if it is greater than the pivot element, then move the right pointer to the left (subtract 1 to the right index). Continue this until right side element is less than or equal to the pivot element.
+6. Check if left pointer is less than or equal to right pointer, then saw the elements in locations of these pointers.
+7. Increment the left pointer and decrement the right pointer.
+8. If index of left pointer is still less than the index of the right pointer, then repeat the process; else return the index of the left pointer.
+
+Sample Image Example
+
++ ![quickSort](quickSort.png)
```
Comment (adrianhuebner) [inline javascript/code-challenges/quickSort/BLOG.md:63]: This looks really great, I would just like to see tests added to this!
